lib/Suggestions.js

```diff
@@ -45,6 +45,7 @@ class Suggestions extends React.Component {
           aria-disabled={item.disabled === true}
           onMouseDown={this.onMouseDown.bind(this, item)}
         >
+          {item.prefix ? <span>{item.prefix}</span> : null}
```

Comment:
I think this `<span>` needs a class name so that it may be explicitly targeted for styling. Take a look at the existing properties on the `classNames` prop.

Reply:
Sounds good, I added one! Let me know if you'd prefer different naming (or ordering in the README)

Reply:
Thanks! I'm thinking maybe the class name should be more generic as this component doesn't know anything about whether or not the suggestion is new, only that it has a prefix 🤔 Perhaps `suggestionPrefix` will suit?

Reply:
Sounds good, updated!